src/components/ClickCatcher.js

```diff
@@ -17,11 +19,20 @@ const ClickCatcher = ({ tag, ...props }, { dispatch }) => {
       node.setAttribute('target', '_blank')
     }
   }
+
+  const handleMouseOver = event => {
+    var node = event.target
+    if (node.nodeName.toLowerCase() === 'a' && node.getAttribute('class') === 'hashtag') {
+      let { tagName, slug } = tagUrlComponents(node.getAttribute('href'))
+      dispatch(showTagPopover(tagName, slug, positionInViewport(node), node.offsetWidth))
```

Comment:
should work the same with `position` if you drop `window.pageYOffset` in the component